tests/DpAccordion.spec.js

```diff
@@ -0,0 +1,58 @@
+import { shallowMount } from '@vue/test-utils'
+import DpAccordion from '~/components/DpAccordion/DpAccordion.vue'
+import DpTransitionExpand from '~/components/DpTransitionExpand'
+
+describe('DpAccordion', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMount(DpAccordion, {
+      props: {
+        dataCy: 'accordionToggle',
+        fontWeight: 'bold',
+        compressed: false,
+        isOpen: false,
+        title: 'Test Title'
+      }
+    })
+  })
+
+  it('renders component correctly', () => {
+    expect(wrapper.exists()).toBe(true)
+  })
+
+  it('renders with the correct title and default props', () => {
+    expect(wrapper.find('button').text()).toContain('Test Title')
+    expect(wrapper.find('button').classes()).toContain('weight--bold')
+    expect(wrapper.find('button').attributes('aria-expanded')).toBe('false')
+    expect(wrapper.find('button').attributes('data-cy')).toContain('accordionToggle')
+  })
+
+  it('toggles the accordion visibility when the button is clicked', async () => {
+    expect(wrapper.vm.isVisible).toBe(false)
+    await wrapper.find('button').trigger('click')
+    expect(wrapper.vm.isVisible).toBe(true)
+    expect(wrapper.find('button').attributes('aria-expanded')).toBe('true')
```

Comment:
In my opinion, this should be in its own 'it' statement again:

```
it('toggles the value of the aria-expanded attribute when the button is clicked', async () => {
  expect(wrapper.find('button').attributes('aria-expanded')).toBe('false')
  await wrapper.find('button').trigger('click')
  expect(wrapper.find('button').attributes('aria-expanded')).toBe('true')
}
```

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/931/commits/39bda26351732a41df2e0d91e8bc265648cb09fa